cdm/core/src/main/java/ucar/nc2/internal/iosp/hdf5/H5headerNew.java

```diff
@@ -177,10 +178,23 @@ public static boolean isValidFile(RandomAccessFile raf) throws IOException {
   private PrintWriter debugOut;
   private MemTracker memTracker;
 
+  private final Charset valueCharset;
+
   H5headerNew(RandomAccessFile myRaf, Group.Builder root, H5iospNew h5iosp) {
     this.raf = myRaf;
     this.root = root;
     this.h5iosp = h5iosp;
+    valueCharset = h5iosp.getValueCharset().orElse(StandardCharsets.UTF_8);
+  }
+
+  /**
+   * Return defined {@link Charset value charset} that
+   * will be used by reading netcdf file.
```

Comment:
The reference to netcdf file here isn't clear. In this case, it's the `Charset` used when reading string values out of the hdf5 header, right?